### PR TITLE
add change candidate limits script

### PR DESCRIPTION
**Pull request by joshuahannan**
Adds a template for updating the candidate node limits for each node type

I tested with a script on mainnet to confirm that it works properly and it passed

### CODE REVIEW

File: transactions/set-candidate-limits/2023/nov-3/arguments-change-candidate-limits.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        "type": "Dictionary",
+        "value": [
+            {
+                "key": {
+                    "type": "UInt8",
+                    "value": "1"
+                },
+                "value": {
+                    "type": "UInt64",
+                    "value": "100"
```
Comment (vishalchangrani) [inline transactions/set-candidate-limits/2023/nov-3/arguments-change-candidate-limits.json:12]:
```suggestion
                    "value": "200"
```

File: transactions/set-candidate-limits/2023/nov-3/arguments-change-candidate-limits.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                }
+            },
+            {
+                "key": {
+                    "type": "UInt8",
+                    "value": "2"
+                },
+                "value": {
+                    "type": "UInt64",
+                    "value": "100"
```
Comment (vishalchangrani) [inline transactions/set-candidate-limits/2023/nov-3/arguments-change-candidate-limits.json:22]:
```suggestion
                    "value": "200"
```

Comment (vishalchangrani) [reply transactions/set-candidate-limits/2023/nov-3/arguments-change-candidate-limits.json:22]: Consensus we need 200

File: transactions/set-candidate-limits/2023/nov-3/arguments-change-candidate-limits.json (hunk 1 above)
Comment (vishalchangrani) [reply transactions/set-candidate-limits/2023/nov-3/arguments-change-candidate-limits.json:12]: Collection 200

File: transactions/set-candidate-limits/2023/nov-3/arguments-change-candidate-limits.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                }
+            },
+            {
+                "key": {
+                    "type": "UInt8",
+                    "value": "3"
+                },
+                "value": {
+                    "type": "UInt64",
+                    "value": "10"
```
Comment (vishalchangrani) [inline transactions/set-candidate-limits/2023/nov-3/arguments-change-candidate-limits.json:32]:
```suggestion
                    "value": "20"
```

Comment (vishalchangrani) [reply transactions/set-candidate-limits/2023/nov-3/arguments-change-candidate-limits.json:32]: Execution node 20 (instead of 10)

Comment (vishalchangrani) [reply transactions/set-candidate-limits/2023/nov-3/arguments-change-candidate-limits.json:32]: commiting the change @joshuahannan (hope you don't mind)

**Review by bjartek — APPROVED** [review]

**Review by vishalchangrani — APPROVED** [review]

**Pull request merged**